src/client/src/rt-platforms/browser/window.ts

```diff
@@ -5,7 +5,7 @@ import { WindowConfig } from '../types'
 type BrowserWindowProps = WindowConfig
 
 export const openBrowserWindow = function(
-  this: { prevWindow: Window },
+  this: { prevWindow: Window; externalWindows: Window[] },
```

Comment:
if you brought the `this.externalWindows = this.externalWindows.filter(window => !window.closed)` you could remove the `prevWindow` altogether I think, and use `this.externalWindows[this.externalWindows.length -1]` instead

Reply:
I used the `prevWindow` for the `left` and `top` properties in the `windowReferencePosition` object so I think it's still a good idea to keep it as a separate variable.  However, I did update the function to use the last element in the array for the `prevWindow`